analysis/tidy_measures_med_counts.R

```diff
@@ -0,0 +1,24 @@
+library(here)
+library(tidyverse)
+library(readr)
+library(gt)
+library(purrr)
+
+df <- read_csv(here("output", "measures", "pf_medications_measures.csv"))
+df1 <- read_csv(here("output", "measures", "consultation_med_counts_measures.csv"))
```

Comment:
I think it would be clearer if we use names that describe the data here?

```suggestion
df_pf_medications_measures <- read_csv(here("output", "measures", "pf_medications_measures.csv"))
df_consultation_med_counts_measures <- read_csv(here("output", "measures", "consultation_med_counts_measures.csv"))
```

Reply:
I got the following error when I used the existing files as outputs in the .yaml:

```
=> ProjectValidationError
   Invalid project: 
   Output path output/measures/pf_medications_measures.csv is not unique
```